sdk/python/kfp/registry/registry_client_test.py

```diff
@@ -23,23 +24,34 @@
 from kfp.registry import RegistryClient
 
 _DEFAULT_HOST = 'https://us-central1-kfp.pkg.dev/proj/repo'
+_KFP_CONFIG_FILE = os.path.join(
```

Comment:
Is the `./context/` directory only for testing?

If so, I think using `tempdir` to create a temporary directory via the `setUpClass` and `tearDownClass` class methods could be somewhat cleaner. It's possible I'm missing something here, however.

Example of this pattern: https://github.com/kubeflow/pipelines/blob/08e9c171f11477ce72ca3c9296610437ad4ca824/sdk/python/kfp/components/component_factory_test.py#L174-L194

Reply:
On second thought: having this as a sample is nice, though perhaps we don't need to have a separate directory for it. WDYT?

Reply:
The `./context` directory is to provide default config files for known hosts, so that users don't have to specify config files for those (e.g. `*-kfp.pkg.dev`).
Users can also specify a location for a custom config file if they'd like, during the initialization of `RegistryClient`.